massa-execution-worker/src/tests/mod.rs

```diff
@@ -0,0 +1,3 @@
+// Copyright (c) 2022 MASSA LABS <info@massa.net>
+
+//TODO mod scenarios_mandatories;
```

Comment:
Why did you commented out the tests ?

Reply:
because the rewrite broke them, I will reeenable them as a followup, but I decided to wait a bit given the collision with https://github.com/massalabs/massa/pull/2296